test/pco_api/people/workflow_test.exs

```diff
@@ -73,20 +73,28 @@ defmodule PcoApi.People.WorkflowTest do
     %PcoApi.Record{id: "1000", links: %{}} |> Workflow.self
   end
 
-  test ".cards returns a list of workflow cards", %{bypass: bypass} do
+  test ".cards gets a list of cards with a cards link", %{bypass: bypass} do
     Bypass.expect bypass, fn conn ->
+      assert "/people/v2/workflows/1/cards" == conn.request_path
       Plug.Conn.resp(conn, 200, Fixture.read("workflow_cards.json"))
     end
-    workflow = %PcoApi.Record{id: "1", links: %{"cards" => "https://api.planningcenteronline.com/people/v2/workflows/1/cards"}}
-    assert [%PcoApi.Record{type: "WorkflowCard"} | rest] = workflow |> Workflow.cards
+    record_with_link |> Workflow.cards
   end
 
-  test ".steps returns a list of workflow steps", %{bypass: bypass} do
+  test ".steps gets a list of steps with a steps link", %{bypass: bypass} do
     Bypass.expect bypass, fn conn ->
+      assert "/people/v2/workflows/1/steps" == conn.request_path
       Plug.Conn.resp(conn, 200, Fixture.read("workflow_steps.json"))
     end
-    workflow = %PcoApi.Record{id: "1", links: %{"steps" => "https://api.planningcenteronline.com/people/v2/workflows/1/steps"}}
-    assert [%PcoApi.Record{type: "WorkflowStep"} | rest] = workflow |> Workflow.steps
```

Comment:
Ha, I think the diff makes it look like I did something I didn't. I replaced the _entire_ test file trying to emulate what was done in people_test and address_test. We can add that assertion in but that assertion currently isn't happening in the Person or Address tests, other than lines 27 & 36 in both this test and person_test.exs.